tests/test_schema.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework_json_api.schemas.openapi import AutoSchema
```

Comment:
Great that you added a test to our new tests folder.

What we try to do here is that the test file is in the same module structure as the python file it is testing.

So this file should be `tests/schemas/test_openapi.py`

Reply:
Moved.